### DISCUSSION TITLE
[Art competition] Testing

### DISCUSSION THREAD

**Discussion by pencaty** [prosyslab-classroom/cs348-information-security#415] [Art competition]
![white_recs_in_black_space](https://github.com/prosyslab-classroom/cs348-information-security/assets/29794289/98f60559-e736-41d5-8384-9548f35c3770)

## Short Description
For me, [Unsound Testing] means [There are several white rectangles on a black background]".

## Long Description (up to 3 sentences)
As Dijkstra said, program testing does not prove the absence of bugs. Testing executes finite number of input, indicating there are still uncovered branches.

**Comment by KihongHeo** [comment]
Good. It would be nice if the drawing reflects more unique features of testing.
